phaseshifts/atorb.py

```
@@ -858,3 +1255,358 @@ def calculate_Q_density(**kwargs):
             if output_dir is not None
             else output_filename
         )
+
+
+class EEASiSSSAtorb(Atorb):
+    def __init__(self, ifil=0, **kwargs):
```

Ruff (0.14.10)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)